edgelet/contrib/config/windows/config.yaml

```diff
@@ -160,7 +160,7 @@ agent:
   type: "docker"
   env: {}
   config:
-    image: "mcr.microsoft.com/azureiotedge-agent:1.0.9"
+    image: "mcr.microsoft.com/azureiotedge-agent:1.0"
```

Comment:
Did we test this? Might be better to keep this 1.0.9.1 (to avoid unnecessary changes)

Reply:
Dylan did the test with me and it shows that a new container will get the latest version.  Also Edge agent will re-deploy if it detects different version registered in azure portal.  I think it should be safe to make this change.